codespell_lib/data/dictionary.txt

```diff
@@ -17045,6 +17055,7 @@ ligh->light, lie, lye,
 ligher->lighter, liar, liger,
 lighers->lighters, liars, ligers,
 lightweigh->lightweight
+lightwieght->lightweight
```

Comment:
I suspect it probably is, but could you just check that `wieght` is already present as a typo.

Reply:
it is not!  added it!

Reply:
Do you fancy adding wieghts and wieghted too?

Reply:
sure, added

Reply:
Thanks, and good thought with weighed!